cdap-hbase-spi/src/main/java/co/cask/cdap/spi/hbase/HBaseDDLExecutor.java

```diff
@@ -64,7 +70,10 @@ void createTableIfNotExists(TableDescriptor descriptor, @Nullable byte[][] split
     throws IOException;
 
   /**
-   * Enable the specified table if it is disabled.
+   * Enable the specified table if it is disabled. Changing the dataset specs results into
```

Comment:
CDAP does not call it, but implementation of `truncate` should disable the table https://github.com/caskdata/cdap/blob/release/4.2/cdap-hbase-compat-base/src/main/java/co/cask/cdap/data2/util/hbase/DefaultHBaseDDLExecutor.java#L202. 
Updated javadoc for the truncate method to add this.